pkg/podmounter/mppod/mppod_test.go

```diff
@@ -0,0 +1,55 @@
+package mppod_test
+
+import (
+	"testing"
+
+	"github.com/google/uuid"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/awslabs/aws-s3-csi-driver/pkg/podmounter/mppod"
+	"github.com/awslabs/aws-s3-csi-driver/pkg/util/testutil/assert"
+)
+
+func TestGeneratingMountpointPodName(t *testing.T) {
+	t.Run("Consistency", func(t *testing.T) {
+		pod := &corev1.Pod{
+			ObjectMeta: metav1.ObjectMeta{UID: types.UID(uuid.New().String())},
+		}
+		pvc := &corev1.PersistentVolumeClaim{
+			Spec: corev1.PersistentVolumeClaimSpec{VolumeName: "test-vol"},
+		}
+
+		assert.Equals(t,
```

Comment:
Maybe add a comment here saying the test is that the function returns the same thing when called multiple times, as it tripped me up the first read through

Reply:
I thought test name "Consistency" would make it clear, but I can also add some comments on top of it

Reply:
Added comment with https://github.com/awslabs/mountpoint-s3-csi-driver/pull/299/commits/229a7517380f10d969b50fc0a17ea2ad6ea9fffa